fury/ui/tests/test_elements.py

```diff
@@ -1124,6 +1124,37 @@ def test_ui_draw_panel_rotation(interactive=False):
         event_counter.check_counts(expected)
 
 
+def test_ui_draw_panel_grouping(interactive=False):
+    filename = "test_ui_draw_panel_grouping"
+    recording_filename = pjoin(DATA_DIR, filename + ".log.gz")
+    expected_events_counts_filename = pjoin(DATA_DIR, filename + ".json")
+
+    drawpanel = ui.DrawPanel(size=(600, 600), position=(30, 10))
+
+    # Assign the counter callback to every possible event.
+    event_counter = EventCounter()
+    event_counter.monitor(drawpanel)
+
+    current_size = (680, 680)
+    show_manager = window.ShowManager(
+        size=current_size, title="DrawPanel Grouping UI Example")
```

Comment:
You do not need initialize here?

Reply:
Nope, it is internally called in the `show_manager.start()`